RNTester/js/PickerExample.js

```diff
@@ -13,115 +13,119 @@
 const React = require('react');
 const ReactNative = require('react-native');
 const StyleSheet = require('StyleSheet');
-const RNTesterBlock = require('RNTesterBlock');
-const RNTesterPage = require('RNTesterPage');
 
 const {Picker, Text} = ReactNative;
 
 const Item = Picker.Item;
 
-class PickerExample extends React.Component<{}, $FlowFixMeState> {
-  static title = '<Picker>';
-  static description =
-    'Provides multiple options to choose from, using either a dropdown menu or a dialog.';
+type State = {
+  value: string,
+};
 
+class BasicPickerExample extends React.Component<{}, State> {
```

Comment:
Also needed to refactor this to export multiple examples

Reply:
Cool, having all the example modules be more consistent will also be helpful for new contributors.